Source/WebCore/Modules/Model/InternalAPI/DDTextureDescriptor.h

```diff
@@ -25,7 +25,7 @@
 
 #pragma once
 
-#include "DDImageAsset.h"
+#include <WebCore/DDImageAsset.h>
```

Comment:
@mwyrzykowski this will soon be enforced at compile time, but in the meantime if you could remember to try to use angle includes instead of quoted includes for any new imports you add, that will be really appreciated!! thanks!